packages/python/build

```diff
@@ -18,6 +18,12 @@ rm -rf "$PKG_PATH/lib/python3.4/test"
 # mkpanda option to apply to any package.
 find "$PKG_PATH/lib/" ! -type d -name "*.a" -exec rm -f -- '{}' +
 
+# This works around the issue of Python's build not being able to hardlink these files.
+# This is due to the build being performed inside of the VM's shared folder.
+# https://www.virtualbox.org/ticket/818
+if [[ "$OS" == "darwin" ]]; then
+    cp -f "$PKG_PATH/bin/python3.4m" "$PKG_PATH/bin/python3.4"
```

Comment:
I am using Docker on Mac and I had not dependency on Virtualbox (Actually the comment referencing virtualbox surprised me a bit).

* For OSX, I think, it is safe to assume that customers will be using "Docker on Mac", and if we target that, then this `code-stanza` is not required.